### DISCUSSION TITLE
Missing stats.log

### DISCUSSION THREAD

**Discussion by rlefever** [Security-Onion-Solutions/securityonion#1679] [Unsupported Versions] [answered]
I just setup nullmailer on SO 16.04 and updated. I am now getting messages evey5 minutes stating:

error reported by stats-to-csv
manager ...
Error: [Errno 2] No such file or directory: '/nsm/bro/spool/stats.log'

failed to append file: [Errno 2] No such file or directory: '/nsm/bro/spool/stats.log'

-- 
[Automatically generated.]

I created the file and the message did not return for one cycle, but the file was removed and the messages returned.
Seems the cron job is running a script looking for the file but what is supposed to generate the file in the first place? There are stats.log files in other places, just not where it is looking.

I ask the Zeek community and was told to ask here.

Any and all assistance is appreciated.

**Comment by dougburks** [comment, marked as answer]
Hi @rlefever ,

When I responded to you on the Zeek mailing list, I said:
```
Security Onion questions are probably best handled via the Security Onion mailing list:
https://docs.securityonion.net/en/16.04/mailing-lists.html
```

That page should have pointed you to the Security Onion 16.04 mailing list at:
http://groups.google.com/group/security-onion

This Discussions page is for the new Security Onion 2 only.  Please post Security Onion 16.04 questions on the Security Onion 16.04 mailing list as described above.

Thanks!

**Comment by rlefever** [comment]
My mistake. I went to the Security Onion site, clicked support and chose Community.Thanks for the clarification.On Oct 27, 2020 11:54 AM, Doug Burks <notifications@github.com> wrote:
Hi @rlefever ,
When I responded to you on the Zeek mailing list, I said:
Security Onion questions are probably best handled via the Security Onion mailing list:
https://docs.securityonion.net/en/16.04/mailing-lists.html

That page should have pointed you to the Security Onion 16.04 mailing list at:
http://groups.google.com/group/security-onion
This Discussions page is for the new Security Onion 2 only.  Please post Security Onion 16.04 questions on the Security Onion 16.04 mailing list as described above.
Thanks!
